### DISCUSSION TITLE
Step 2: Code with Github Copilot

### DISCUSSION THREAD

**Discussion by PromptStomp** [skills/.github#310] [Code with Copilot] [answered]
I'm not sure what to do next? please help

This is where I got stuck!

keyboard: Activity: Push code to your repository from the codespace
Use the VS Code terminal to add the skills.js file to the repository:

git add skills.js
Next from the VS Code terminal stage and commit the changes to the repository:

git commit -m "Copilot first commit"
Finally from the VS Code terminal push to code to the repository:

git push
Wait about 60 seconds then refresh your repository landing page for the next step.

**Comment by sinsukehlab** [comment, marked as answer]
Create a file named "`skills.js`" and run `git add skills.js` on the VS Code terminal (the panel at the bottom of the page) to stage the file.
Read the [Pro Git](https://git-scm.com/book/en/v2) book if you are unfamiliar with Git.
Take the course "Introduction to GitHub" if you are unfamiliar with GitHub.
Take the course "Code with Codespaces" if you don't know how to use Codespaces.

**Reply by altostec** [reply]
I did the same step and after 60 seconds, and more, and after refresh many times, nothing happened. How to get the next step?

**Reply by sinsukehlab** [reply]
@altostec The workflow has run and an error has occurred. Check the details on the Actions tab of your repo.
See: #304

**Reply by AddyD20** [reply]
From where can i learn "Introduction to GitHub and Code with codespaces??

**Reply by sinsukehlab** [reply]
@AddyD20 You're now in the Discussions tab. Go to the Repositories tab to view the skills organization's repositories. Select a course such as "Introduction to GitHub" and follow the course instructions.
There's also a special page: https://skills.github.com/
